test/functional/feature_finalizer.py

```diff
@@ -12,12 +12,13 @@
     assert_equal,
     assert_finalizationstate,
     connect_nodes,
+    generate_block,
     sync_blocks,
 )
 
 
 def generate_block(node):
-    node.generatetoaddress(1, node.getnewaddress('', 'bech32'))
+    generate_block(node)
```

Comment:
I think we can remove the outer function

Reply:
omg, we **must** actually, it makes infinite recursion. I did fix it actually, but forget to include in the final commit :(